resource-managers/kubernetes/core/src/test/scala/org/apache/spark/deploy/k8s/KubernetesVolumeUtilsSuite.scala

```diff
@@ -148,7 +160,7 @@ class KubernetesVolumeUtilsSuite extends SparkFunSuite {
       KubernetesNFSVolumeConf("/share", "nfs.example.com"))
   }
 
-  test("Fails on missing path option") {
+  test("Fails on missing path option in nfs") {
```

Comment:
ditto. Could you revert this irrelevant change, please?

Reply:
done